core/client/src/main/java/alluxio/hadoop/AbstractFileSystem.java

```diff
@@ -396,15 +397,15 @@ public void initialize(URI uri, org.apache.hadoop.conf.Configuration conf) throw
     Preconditions.checkNotNull(uri.getPort(), PreconditionMessage.URI_PORT_NULL);
 
     super.initialize(uri, conf);
-    LOG.info("initialize({}, {}). Connecting to Alluxio: {}", uri, conf, uri.toString());
+    LOG.info("initialize({}, {}). Connecting to Alluxio", uri, conf);
     HadoopUtils.addS3Credentials(conf);
     HadoopUtils.addSwiftCredentials(conf);
     setConf(conf);
     mAlluxioHeader = getScheme() + "://" + uri.getHost() + ":" + uri.getPort();
     // Set the statistics member. Use mStatistics instead of the parent class's variable.
     mStatistics = statistics;
     mUri = URI.create(mAlluxioHeader);
-    LOG.info("{} {}", mAlluxioHeader, mUri);
+    LOG.debug("{} {}", mAlluxioHeader, mUri);
```

Comment:
I think the URI will be the header plus any additional directory path, right?

Right now, the debug message is pretty useless. We should either remove it, or change it to something like: `header: {} uri: {}`